ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -358,32 +357,32 @@ open class PillButtonBar: UIScrollView {
     ///  portion of it visible and there's a clear indication that the view is scrollable. To achieve this, this function calculates
     ///  a new spacing and padding that will shift the last visible button farther in the view.
     private func adjustButtonsForCurrentScrollFrame() {
-        var visibleWidth = frame.width - (Constants.minButtonsSpacing + Constants.minButtonVisibleWidth)
-        var visibleButtonsWidth = Constants.sideInset
+        var visibleWidth = frame.width - (pillButtonBarTokens.minButtonsSpacing + pillButtonBarTokens.minButtonVisibleWidth)
+        var visibleButtonsWidth = pillButtonBarTokens.sideInset
         var visibleButtonCount = 0
         for button in buttons {
             button.layoutIfNeeded()
-            visibleButtonsWidth += button.frame.width
+            visibleButtonsWidth! += button.frame.width
```

Comment:
I will think on the logic that makes the most sense here. I hesitate to default to a value of 16 because I generally prefer not to use magic numbers (especially since the purpose of this change is to move the definition of all of these attributes/values/tokens into the token system, rather than hard coded).

Reply:
Actually the problem seems to be that `pillButtonBarTokens.sideInset` is declared as a `CGFloat!`, which confuses the auto-typer sometimes. If you explicitly name the tyupe of `visibleButtonsWidth` on line 361 like so:

```swift
var visibleButtonsWidth: CGFloat = pillButtonBarTokens.sideInset
```

then all is well, and you can remove all the force unwraps (`!`) from here on out.